src/prefixer/ModuleManager.spec.ts

```diff
@@ -1553,31 +1554,86 @@ describe('ModuleManager', () => {
             });
         });
 
-        it('prefixes indirect task reference from component in different module', async () => {
+        it('handles d.bs default param values properly', async () => {
+            await createDependencies([{
+                name: 'alpha',
+                dependencies: [{
+                    name: 'beta',
+                    _files: {
+                        'source/betaLib.d.bs': trim`
+                            sub doSomething(callback = noop)
+                            end sub
+                            sub noop()
+                            end sub
+                        `
+                    }
+                }]
+            }]);
+
+            await managerProcess();
+
+            fsEqual(`${hostDir}/source/roku_modules/beta_v1/betaLib.d.bs`, `
+                namespace beta.v1
+                sub doSomething(callback = beta.v1.noop)
+                end sub
+                end namespace
+                namespace beta.v1
+                sub noop()
+                end sub
+                end namespace
+            `);
+        });
+
+        //TODO eventually we need to fix this test
+        it.skip('handles d.bs default param namespaced values properly', async () => {
```

Comment:
This is another case we should eventually support, but I don't have the time to fix it right now and didn't want to hold up the release. I'll leave this test here as a reminder that we do need to fix it eventually.